aquacomputer_d5next.c

```diff
@@ -850,6 +854,8 @@ static int aqc_get_ctrl_data(struct aqc_data *priv)
 	if (ret < 0)
 		ret = -ENODATA;
 
+	print_hex_dump_bytes("ctrl_report: ", DUMP_PREFIX_OFFSET, priv->buffer, priv->buffer_size);
```

Comment:
As mentioned above, this gets disabled by the normal kernel debug infrastructure, and I found it super useful to be able to enable/disable this at runtime. Do you still think it should be removed?

Reply:
For debugging and RE, it's most welcome, but for the driver in this repo, yes.  It's not needed during normal operation, as 99% of users aren't actively reverse engineering anything and don't expect a driver to pollute the logs. Also, I always strive to minimize the discrepancy between the repo and in-kernel state, and debug messages like this wouldn't fly there.

Reply:
As mentioned, this won't show up in logs unless explicitly enabled (documented [here](https://www.kernel.org/doc/html/v4.19/admin-guide/dynamic-debug-howto.html)). I will of course respect your preference anyway!

Reply:
I know that it's configurable, but dumping the received report is not useful in the context of this driver. For some fickle devices, in case of an error, firmware update during runtime or similar, I'd be OK. But here, hidraw is enabled and so the driver is completely transparent to userspace. No need to dump the received data from within the kernel when it's just going to be parsed normally anyway.

Reply:
Got it :+1: